.github/workflows/check-attributions.yml

```diff
@@ -3,13 +3,6 @@ name: Check Attributions
 on:
   push:
     branches: Version-v*
-  pull_request:
-    branches: Version-v*
```

Comment:
`branches` here refers to the target branch, rather than the HEAD branch. That's why this was running on cherry pick PRs.

See https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request